### PR TITLE
changes made

### PR DESCRIPTION
**Pull request by abrolnalin**

### CODE REVIEW

File: lib/resources/powerpacks.js [hunk 1]
```diff
@@ -108,17 +114,42 @@ export class Powerpack extends PlivoResource {
 		let path = 'NumberPool/' + this.number_pool_id + '/Number/' + number + '/';
 		return super.customexecuteAction(path.toString().trim(), 'DELETE', params);
 	}
+	remove_tollfree(tollfree, unrent = false) {
+	var params = {};
+		if (typeof unrent === 'boolean') {
```
Comment (narayana-plivo) [inline lib/resources/powerpacks.js]: you expecting unrent = false in function arg, why do you need type check for unrent?

File: lib/resources/powerpacks.js [hunk 2]
```diff
@@ -312,6 +346,44 @@ export class Shortcode extends PlivoResource {
...
+	}
+	add(tollfree) {
+		var params = {};
+		params['rent'] = 'false';
+		let path = 'NumberPool/' + this.number_pool_id + '/Tollfree/' + tollfree + '/';
+		return super.customexecuteAction(path.toString().trim(), 'POST', params);
+	}
+	remove(tollfree, unrent = false) {
+		var params = {};
+		if (typeof unrent === 'boolean') {
```
Comment (narayana-plivo) [inline lib/resources/powerpacks.js]: same here

File: test/powerpacks.js [hunk 3]
```diff
@@ -87,5 +87,60 @@ describe('PowerpackInterface', function () {
...
+  it('remove tollfree via interface', function () {
+    client.powerpacks.get("5ec4c8c9-cd74-42b5-9e41-0d7670d6bb46").then(
+        function (powerpack) {
+          return powerpack.remove_tollfree("18772209942", true)
+        })
+      .then(function (result) {
+        assert.notEqual(result.response, "success")
+      })
+  });
+  it('remove shortcode via interface', function () {
```
Comment (narayana-plivo) [inline test/powerpacks.js]: it should be remove tollfree via interface

File: test/powerpacks.js [hunk 4]
```diff
@@ -87,5 +87,60 @@ describe('PowerpackInterface', function () {
...
+  it('add tollfree to powerpack via interface', function () {
+    client.powerpacks.get("5ec4c8c9-cd74-42b5-9e41-0d7670d6bb46").then(
+        function (powerpack) {
+          return powerpack.add_tollfree('18772209942')
+        })
+      .then(function (result) {
+        assert.Equal(result.number, "18772209942")
+      })
+  });
+  it('remove tollfree via interface', function () {
```
Comment (narayana-plivo) [inline test/powerpacks.js]: looks like duplicate

File: lib/resources/powerpacks.js (hunk 1 above)
Comment (abrolnalin) [reply lib/resources/powerpacks.js]: changed

File: test/powerpacks.js (hunk 4 above)
Comment (abrolnalin) [reply test/powerpacks.js]: removed duplicate

**Pull request merged**